packages/pwa/pwa-kit-config.json

```diff
@@ -1,7 +1,61 @@
 {
     "app": {
+        "defaultSiteId": "RefArchGlobal",
         "url": {
-            "locale": "path"
-        }
+            "locale": "path",
+            "site": "path"
+        },
+        "sites": [
+            {
+                "id": "RefArch",
+                "alias": "us",
+                "l10n": {
+                    "supportedCurrencies": ["USD"],
+                    "defaultCurrency": "USD",
+                    "supportedLocales": [
+                        {
+                            "id": "en-US",
+                            "preferredCurrency": "USD"
+                        },
+                        {
+                            "id": "en-CA",
+                            "preferredCurrency": "USD"
+                        }
+                    ],
```

Comment:
Are we going to be supporting aliasing of the local short-code as well? It might be useful if a customer was to want a url that looked like `http://www.example.com/ca/en/categories/hats`.

Reply:
I'd say it is a nice-to-have for the scope of this PR. To avoid making this PR bigger than it should, we can work on the locale alias after this PR